src/logic/operate.js

```diff
@@ -0,0 +1,34 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result;
+  try {
+    const one = Big(+numberOne);
+    const two = Big(+numberTwo);
+
+    if (operation === '÷') {
+      result = Big(one.div(two));
+    }
+
+    if (operation === '+') {
+      result = Big(one.plus(two));
+    }
+
+    if (operation === 'X') {
+      result = Big(one.times(two));
+    }
+
+    if (operation === '-') {
+      result = Big(one.minus(two));
+    }
+
+    if (operation === '%') {
+      result = Big(one.div(two));
```

Comment:
Kindly use `one.mod(100)` to convert any number into a percentage instead `one.div(two)` the logic is not right